src/tools/assistant/search_knowledge.ts

```diff
@@ -0,0 +1,68 @@
+import { z } from "zod";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types.js";
+import { ToolArgs, OperationType } from "../tool.js";
+import { AssistantToolBase } from "./assistantTool.js";
+
+export const SearchKnowledgeToolArgs = {
+    query: z.string().describe("A natural language query to search for in the knowledge base"),
+    limit: z.number().min(1).max(100).optional().default(5).describe("The maximum number of results to return"),
```

Comment:
```suggestion
    limit: z.number().min(1).max(100).optional().default(10).describe("The maximum number of results to return"),
```

i've seen 10 chunks in a bunch of other implementations. we dont do this in our chatbot, but that's largely for latency/price